UpdateRCConfiguration/handler.ts

```diff
@@ -49,6 +63,17 @@ export const handleUpsert = (rccModel: RCConfigurationModel) => (
         `Something went wrong trying to upsert the configuration: ${e}`
       )
     ),
+    TE.chainFirst(newConfiguration =>
```

Comment:
Maybe it would be better to call this `rcConfiguration` only, because the prefix `new` is more suitable for a create operation than for an update.

Reply:
[30d3692](https://github.com/pagopa/io-functions-service-messages/pull/72/commits/30d3692c91635da84e1e66b60e85878abaa6bfd1)